src/thunderbots/software/util/ros_messages.cpp

```diff
@@ -180,5 +178,62 @@ namespace Util
             World world(field, ball, friendly_team, enemy_team);
             return world;
         }
+
+        thunderbots_msgs::World transformWorldMessage(
+            const thunderbots_msgs::World& old_world_msg)
+        {
+            thunderbots_msgs::World new_world_msg;
```

Comment:
make the `new_world_msg` a copy of the `old_world_msg` initially, so we only transform and overwrite data we need to. Otherwise the new `world_msg` we return is missing information.